Documentation/ExtensionInstallation/Index.rst

```diff
@@ -55,6 +63,86 @@ In the backend:
 8. Click on the icon with a :guilabel:`"+"` sign for your extension
    in the :guilabel:`"A/D"` column.
 
+.. _uninstall_extension_without_composer
+
+Uninstall an Extension Without Composer
+=======================================
+
+If you installed TYPO3 via composer you should uninstall Extensions via composer.
+
+Check Dependencies
+------------------
+
+First find out, which other extensions and functions of your TYPO3-Installation are dependent on the extension you
+want to deinstall. You can find out about the dependencies by checking the
+`Extension Repository <https://extensions.typo3.org/>`__. Look for the extension
+you want to deinstall and the others you have installed. Read in each extensions manual the sections 'Dependencies' and
+'Reverse dependencies'.
+
+Contemplate, whether you made referrals to the extension in any setup, config or other TypoScript-section. Check if you
+included plugins from the extension in your page. Think of the results of deleting them and finally do it.
+
+If you are working locally or on a test server you might as well try to deinstall the extension. The Extension Manager
+warns you about dependencies that are written in an extensions :file:`ext_emconf.php` constrains section. Note however
+that you depend on the extensions developers faithfully noting all dependencies in this config file.
+
+If you get an exception and can't even access the Extension Manager anymore because of it, you can deinstall / install
+extensions manually with :file:`PackageStates.php` as a last resort, see :ref:`install-extension-manually`
+
+.. tip::
+   Be sure not to uninstall extensions by trial and error on life systems, expecially not under time pressure.
+
+.. _uninstall-extension-backend:
+
+Uninstall / Deinstall / Remove Extension via TYPO3 Backend
+----------------------------------------------------------
+
+Log into the TYPO3 Backend and open the Extension Manager ('Ext Manager'). From the menu choose 'Install extensions'.
+You get an overview about installed extensions.
+
+On the left side you see an icon, which shows the status of each extension, and what you can do:
+
+* Grey Extension Install Icon: The extension is not installed. (Click once to install)
+* Green Extension UnInstall Icon: The extension is installed and running. (Click once to uninstall)
+
+Next to the extension you want to deinstall click on Extension UnInstall Icon. After some seconds the icon changes to
+the grey Extension Install Icon.
+
+.. _install-extension-manually:
+
+Install / Uninstall an Extension Manually
+----------------------------
```

Comment:
The underline must be at least as long as the header. Otherwise this might break the rendering. I tested the change locally and it renders ok, but this may or may not be a problem in the future, so I would stick with the recommendations.

See https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CommonPitfalls/Headers.html